### PR TITLE
Issue 3 coding standards

### PR DESCRIPTION
**Pull request by dnaber-de**
Changes:
* Updated code formatting to latest inpsyde/php-coding-standard #3
* BREAKING: Added parameter type hints to `LogzIoHandler::__construct(..., int $level, ...)` and `LogzIoHandler::send(string $data)` according to #3

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,12 +18,13 @@
...
-    "monolog/monolog": "^1.23"
+    "monolog/monolog": "^1.23",
+    "ext-curl": "*",
+    "inpsyde/php-coding-standards": "@stable"
   },
   "require-dev": {
     "php": ">=7",
-    "phpunit/phpunit": "6.3.*",
-    "inpsyde/php-coding-standards": "^0.8"
+    "phpunit/phpunit": "6.3.*"
```
Comment (bueltge) [inline composer.json:27]: Why not more globally `^6` to use the last version from 6* ?

Comment (Chrico) [reply composer.json:27]: I think we can even use phpunit 7, since we've PHP 7.x as requirement.

**Comment by OskarStark** [comment]
This one can be closed imo, most of the changes were done in #6 by me

**Comment by Chrico** [comment]
✔️ and thanks for feedback!

**Pull request closed without merging**